src/pod/personal-storage.ts

```diff
@@ -16,25 +23,72 @@ export class PersonalStorage {
    */
   async list(): Promise<Pod[]> {
     assertActiveAccount(this.accountData)
-    let result: Data
+    let data: Data
     try {
-      result = await getFeedData(this.accountData.connection.bee, POD_TOPIC, this.accountData.wallet!.address)
+      data = (
+        await getFeedData(
+          this.accountData.connection.bee,
+          POD_TOPIC,
+          prepareEthAddress(this.accountData.wallet!.address),
+          this.accountData.connection.options?.downloadOptions,
+        )
+      ).data.chunkContent()
     } catch (e) {
       return []
     }
 
-    return result
-      .text()
-      .split('\n')
-      .filter(item => Boolean(item.trim()))
-      .map(item => {
-        const parts = item.split(',')
+    return extractPods(data)
+  }
+
+  /**
+   * Creates new pod with passed name
+   *
+   * @param name pod name
+   */
+  async create(name: string): Promise<Pod> {
+    assertActiveAccount(this.accountData)
+    name = name.trim()
+    let lookupAnswer: LookupAnswer | undefined
+    let list: Pod[]
+    try {
+      lookupAnswer = await getFeedData(
+        this.accountData.connection.bee,
+        POD_TOPIC,
+        prepareEthAddress(this.accountData.wallet!.address),
+        this.accountData.connection.options?.downloadOptions,
+      )
+      list = extractPods(lookupAnswer.data.chunkContent())
+    } catch (e) {
+      list = []
+    }
```

Comment:
extract this snippet into a different function because its logic is identical with the `list`.

Reply:
I have it to be done in one next PR